go.mod

```diff
@@ -171,46 +171,46 @@ require (
 	github.com/wk8/go-ordered-map v0.2.0
 	github.com/xdg-go/scram v1.1.2
 	github.com/zeebo/xxh3 v1.0.2
-	go.opentelemetry.io/collector v0.87.0
-	go.opentelemetry.io/collector/component v0.87.0
-	go.opentelemetry.io/collector/config/configauth v0.87.0
-	go.opentelemetry.io/collector/config/configcompression v0.87.0
-	go.opentelemetry.io/collector/config/configgrpc v0.87.0
-	go.opentelemetry.io/collector/config/confighttp v0.87.0
-	go.opentelemetry.io/collector/config/confignet v0.87.0
-	go.opentelemetry.io/collector/config/configopaque v0.87.0
-	go.opentelemetry.io/collector/config/configtelemetry v0.87.0
-	go.opentelemetry.io/collector/config/configtls v0.87.0
-	go.opentelemetry.io/collector/confmap v0.87.0
-	go.opentelemetry.io/collector/connector v0.87.0
-	go.opentelemetry.io/collector/consumer v0.87.0
-	go.opentelemetry.io/collector/exporter v0.87.0
+	go.opentelemetry.io/collector v0.89.0
```

Comment:
Unfortunately it's not possible to merge this PR, because updating shared OTel dependencies could have a knock-on effect on various OTel components other than the service graph connector. We usually update all OTel dependencies in bulk:
https://github.com/grafana/agent/blob/main/docs/developer/updating-otel.md

Unfortunately it's a very time consuming and error prone process, and maybe not the best candidate for a community contribution :/ I hope the process will be easier in the future.

Reply:
Noprob, thanks for clarifying ✌️